compose.yaml

```diff
@@ -55,7 +55,7 @@ services:
         user: ${MBIN_USER}
 
     redis:
-        image: redis:alpine
+        image: redis:bookworm
```

Comment:
I would love to see valkey here instead of Redis: https://github.com/valkey-io/valkey

Valkey is truly open-source, and is a fork of Redis before they changed their license model. Also Valkey is more performant today.

Reply:
Sure, I wouldn't mind switching it out. Are there any changes I need to be aware of for the migration? (like configuration changes)

Reply:
It's a fully drop in replacement. So no changes are needed. You could however add additional io threads maybe to speedup the current configuration in docker.

We have a very nice website 😬: https://docs.joinmbin.org/admin/configuration/redis

Reply:
Does this look good: https://github.com/MbinOrg/mbin/pull/1542/commits/8506a510b9ab6e19ce5537543a1db62a85aa2638?

Reply:
Yes it does look good. Assuming it also just still works.

Reply:
Yeah, I did test it briefly :)